### DISCUSSION TITLE
can charges be used in hoomd-blue with DPD or (MDPD)  particles?

### DISCUSSION THREAD

**Discussion by rokebymark** [glotzerlab/hoomd-blue#1727] [Support] [answered]
Hi, I am completely new to hoomd-blue and was wanting to ask a little about capabilities. I was wondering firstly if charges can be used with DPD particles (realistically these would have to be smeared charges). Though, if MDPD is available, point charges with MDPD would be OK. Thanks.

**Comment by joaander** [comment, marked as answer]
Yes, you can use charges. HOOMD-blue makes no assumptions about your model, it will add together all forces that you define.
